tools/depends/target/libbluray/Makefile

```diff
@@ -26,6 +26,11 @@ $(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE) $(DEPS)
 	sed -i -e "s|CFLAGS=|#CFLAGS=|" $(PLATFORM)/Makefile.am
 	cd $(PLATFORM); ./bootstrap
 	cd $(PLATFORM); $(CONFIGURE)
+ifeq ($(OS),ios)
+ifeq ($(CPU), arm64)
```

Comment:
Shouldn't this be `ifeq ($(TARGET_PLATFORM),appletvos)`?
If it is also valid for iOS 64 bit please change the patch filename.